### PR TITLE
[TextAnalytics] Added DisableServiceLogs property to ExtractSummaryAction

### PR DESCRIPTION
**Pull request by kinelski**
Fixes https://github.com/azure/azure-sdk-for-net/issues/22799.

Regenerated the code from the latest swagger. This made it possible for us to add the missing `DisableServiceLogs` property to the `ExtractSummaryAction` class.

### CODE REVIEW

**Review by kristapratico — APPROVED** [review]

File: sdk/textanalytics/Azure.AI.TextAnalytics/src/ExtractSummaryAction.cs [hunk 1]
```diff
@@ -30,6 +30,16 @@ public ExtractSummaryAction()
         /// </summary>
         public string ModelVersion { get; set; }
 
+        /// <summary>
+        /// The default value of this property is 'true'. This means, Text Analytics service won't log your input text.
```
Comment (deyaaeldeen) [inline sdk/textanalytics/Azure.AI.TextAnalytics/src/ExtractSummaryAction.cs:34]: I think the default for this is false. It should be true only for actions that process sensitive data such as healthcare and pii.

Comment (kinelski) [reply sdk/textanalytics/Azure.AI.TextAnalytics/src/ExtractSummaryAction.cs:34]: Thank you for pointing this out. Hadn't realized they are different depending on the type of action. Updating it.

Comment (kristapratico) [reply sdk/textanalytics/Azure.AI.TextAnalytics/src/ExtractSummaryAction.cs]: Hmm interesting. The swagger does have this defaulting to True. I'll ask on the PR. https://github.com/Azure/azure-rest-api-specs/blob/d758c4205d331c552cafbb755ed02673b9fa5e22/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.2-preview.1/TextAnalytics.json#L810

**Review by maririos — APPROVED** [review]

File: sdk/textanalytics/Azure.AI.TextAnalytics/src/ExtractSummaryAction.cs (hunk 1 above)
Comment (kinelski) [reply sdk/textanalytics/Azure.AI.TextAnalytics/src/ExtractSummaryAction.cs]: I'll merge this PR to unblock the next ones (keeping it 'false' for now). Created https://github.com/Azure/azure-sdk-for-net/issues/22809 for tracking.

**Pull request merged**